Ginger/GingerCoreNETUnitTest/SolutionTestsLib/RepositorySerializerTest.cs

```diff
@@ -415,5 +415,43 @@ public void BusinessFlowBrowserActionTest()
             Assert.AreEqual("NewTab", (from aiv in businessFlow.Activities[0].Acts[0].InputValues where aiv.Param == "GotoURLType" select aiv).FirstOrDefault().Value);
         }
 
+        
+        [TestMethod]
+        public void ActiviyLazyLad()
+        {
```

Comment:
1. Read XML as string
2. Call DoLazyLoadItems
3. Validate lazy load flag is true
4. Get count and validate
5. Validate lazy load flag is false